core-java-modules/core-java-15/src/main/java/com/baeldung/whatsnew/sealedclasses/Employee.java

```diff
@@ -0,0 +1,13 @@
+package com.baeldung.whatsnew.sealedclasses;
+
+import java.util.Date;
+
+public non-sealed class Employee extends Person {
+
+    public Date getHiredDate()
+    {
```

Comment:
Please put the opening brace at the end of the previous line, not on a line by itself.

Reply:
Done